_i18n/en.yml

```diff
@@ -209,7 +209,8 @@ downloads:
   pruning: Blockchain pruning
   pruning1: Not enough disk space? Just use @pruning to download only 1/3 of the blockchain
   langs: "<b>30+ languages</b> available"
-  cli_intro: The CLI wallet gives you the total control over your Monero node and funds. Highly customizable and includes various analysis tools, as well as an HTTP RPC and 0MQ interface.
+  cli_intro: An open-source command line interface (CLI) wallet developed by the Monero community, completely free to use, best suited for developers, intermediate, and advanced users.
+  cli_intro2: The Monero CLI wallet gives you total control over your node and funds. Highly customizable and includes various analysis tools, as well as an HTTP RPC and 0MQ interface.
```

Comment:
The old placeholder must keep its original name or all translations for that string will be lost.

```suggestion
  cli_intro2: An open-source command line interface (CLI) wallet developed by the Monero community, completely free to use, best suited for developers, intermediate, and advanced users.
  cli_intro: The Monero CLI wallet gives you total control over your node and funds. Highly customizable and includes various analysis tools, as well as an HTTP RPC and 0MQ interface.
```

Reply:
The current code of this PR keeps translation of the first part (see below):
- First part (cli_intro): old spanish translation of cli_intro
- Second part (cli_intro2): previous english sentence (untranslated)

![image](https://user-images.githubusercontent.com/45968869/128171643-14f8fd13-09ec-4dfc-9df5-742c22183270.png)

Reply:
That's true as long as nobody touches the translated string. To weblate it will be an entirely different string, when in reality it just got renamed. That will cause the loss of the translated string for all languages at the next push. The old placeholder must keep its original content + changes, or translators will have to retranslate strings that were already translated even if only a single word has changed (`Monero`) in the string contained by the old placeholder.

Reply:
Ok, so I'll rename the new string to cli_intro0 instead of cli_intro2, because it will be displayed before cli_intro.